src/index.js

```diff
@@ -1,16 +1,34 @@
 import colors from 'colors/safe';
+import _forEach from 'lodash/collection/forEach';
 
 const IS_ERROR_RE = /^error/;
 
 export default function logEvents(cache, options = {}) {
-  cache.onAny(function (...args) {
-    const out = IS_ERROR_RE.test(this.event) ? 'stderr' : 'stdout';
-    const timestamp = (new Date()).toISOString();
-    const log = [colors.grey(timestamp)];
+  if (options.events) {
+    const events = options.events;
+    _forEach(events, function (event) {
+      cache.on(event, function (...args) {
+        const out = IS_ERROR_RE.test(this.event) ? 'stderr' : 'stdout';
```

Comment:
The inner function of `cache.on` and `cache.onAny` is identical. Extract it into a separate function please. Alternatively, you can keep `onAny`, and within it only write the log if the event matches one of the configured events: e.g., `if (events.indexOf(this.event) === -1) return;`